people_recognition_3d/src/people_recognition_3d/people_recognizer_3d.py

```diff
@@ -236,102 +244,96 @@ def callback(self, rgb, depth, depth_info):
-            # markers.markers.append(Marker(header=rgb.header,
-            #                               ns='persons',
-            #                               id=i,
-            #                               type=Marker.CYLINDER,
-            #                               action=Marker.ADD,
-            #                               pose=Pose(position=p_avg, orientation=Quaternion(*q)),
-            #                               scale=Vector3(0.2, 0.2, height),
-            #                               color=ColorRGBA(cmap[i, 0], cmap[i, 1], cmap[i, 2], 0.5)))
-            if "is_pointing" in p.tags:
+                person3d.tags.append("is_pointing")
+                person3d.pointing_pose = pointing_pose
+
+            people3d.append(person3d)
+
+            # visualize persons
+            # height = 1.8
+            # head = 0.2
+
+            # q = tf.transformations.quaternion_from_euler(-math.pi / 2, 0, 0)
+            if "is_pointing" in person3d.tags:
                 markers.markers.append(Marker(header=rgb.header,
                                               ns='pointing_pose',
                                               id=i,
                                               type=Marker.ARROW,
                                               action=Marker.ADD,
-                                              pose=p.pointing_pose,
+                                              pose=person3d.pointing_pose,
                                               scale=Vector3(0.5, 0.05, 0.05),
                                               color=ColorRGBA(cmap[i, 0], cmap[i, 1], cmap[i, 2], 1.0)))
 
-            # markers.markers.append(Marker(header=rgb.header,
-            #                               ns='labels',
-            #                               id=i,
-            #                               type=Marker.TEXT_VIEW_FACING,
-            #                               action=Marker.ADD,
-            #                               lifetime=rospy.Duration(1.5),
-            #                               pose=Pose(position=p_head, orientation=Quaternion(*q)),
-            #                               text=text,
-            #                               scale=Vector3(0, 0, 0.2),
-            #                               color=ColorRGBA(cmap[i, 0], cmap[i, 1], cmap[i, 2], 1)))
-
-        self.person_pub.publish(People(header=rgb.header, people=persons))
+        # self.person_pub.publish(People(header=rgb.header, people=people3d))
         # publish all markers in one go
-        self.markers_pub.publish(markers)
+        # self.markers_pub.publish(markers)
+        rospy.loginfo("Done. Found {} people, {} markers".format(len(people3d), len(markers.markers)))
+        return people3d, markers
 
     def recognitions_to_joints(self, recognitions, rgb, depth, cam_model):
-        cv_depth = self.bridge.imgmsg_to_cv2(depth)
+        cv_depth = self._bridge.imgmsg_to_cv2(depth)
```

Comment:
If you start to rename every variable for no reason, you will loose git gistory. Please don't autopep8 or 'cleanup' code for no reason.